gui/src/3D/gui3DManager.ts

```diff
@@ -185,6 +217,8 @@ export class GUI3DManager implements IDisposable {
      */
     public addControl(control: Control3D): GUI3DManager {
         this._rootContainer.addControl(control);
+        control.scaling.scaleInPlace(this._customControlsScale);
+        control._isScaledByManager = true;

```

Comment:
This is set to true by default, but in the `controlScaling` function it is only true if newScale is not 1. Any side-effects for that?